internal/pkg/docker/dockerfile/dockerfile.go

```diff
@@ -23,6 +26,18 @@ const (
 )
 const reFindAllMatches = -1 // regexp package uses this as shorthand for "find all matches in string"
 
+const (
+	intervalFlag       = "interval"
+	timeoutFlag        = "timeout"
+	startPeriodFlag    = "start-period"
+	retriesFlag        = "retries"
+	intervalDefault    = 30000000000 //30s in nanoseconds
+	timeoutDefault     = 30000000000 //30s in nanoseconds
+	startPeriodDefault = 0
+	retriesDefault     = 3
+	healthcheck        = 12
```

Comment:
Why is it 12 since len(healthcheck) == 11? Also `healthcheck` is not a clear constant name, what do you think of `healthcheckInstructionStartIndex`?

Reply:
That sounds good and it is 12 because of the space after HEALTHCHECK.

Reply:
ooh gotcha, one thing that we can write to make it clearer is like this: https://play.golang.org/p/4rywkjWwk31

```go
const (
   hcInstrStartIndex = len("HEALTHCHECK ")
)
```